pkg/app/logging.go

```diff
@@ -0,0 +1,49 @@
+// +build !windows
+
+package app
+
+import (
+	"fmt"
+	"github.com/aybabtme/humanlog"
+	"github.com/jesseduffield/lazygit/pkg/config"
+	"github.com/jesseduffield/lazygit/pkg/secureexec"
+	"log"
+	"os"
+)
+
+func TailLogs1() {
```

Comment:
is the `1` needed here?

Reply:
o no, this is completely wrong - this was me trying to prove that I'd done conditional compilation properly (ie that it built under windows and definitely used the windowsy TailLogs). Fixed up.